digits/job.py

```diff
@@ -324,6 +325,12 @@ def wait_completion(self):
         if hasattr(self, 'event'):
             self.event.wait()
 
+    def delete_timeout(self):
+        """
+        Returns how long a non-persistent job should exist for
+        """
+        return NON_PERSISTENT_JOB_DELETE_TIMEOUT_SECONDS
```

Comment:
I guess the function is fine, but you can move the variable into the class, I would think.